amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseRefresherTest.java

```diff
@@ -265,7 +266,40 @@ public void testLeaseTableExistsTimesOut() throws Exception {
     }
 
     @Test
-    public void testCreateLeaseTableTimesOut() throws Exception {
+    public void testCreateLeaseTableProvisionedBillingModeIfNotExists() throws Exception {
+        leaseRefresher = new DynamoDBLeaseRefresher(TABLE_NAME, dynamoDbClient, leaseSerializer, CONSISTENT_READS,
+                tableCreatorCallback, LeaseManagementConfig.DEFAULT_REQUEST_TIMEOUT, BillingMode.PROVISIONED);
+
+        when(dynamoDbClient.describeTable(any(DescribeTableRequest.class))).thenReturn(mockDescribeTableFuture);
```

Comment:
I don't think we need to use `any(DescribeTableRequest.class)` since the table name is constant across tests, we can mock against a concrete object like `DescribeTableRequest.builder().tableName(TABLE_NAME).build()`.